config/master.d/master.conf

```diff
@@ -4,6 +4,7 @@ interface: 0.0.0.0
 event_return: mysql
 presence_events: True
 worker_threads: 10
+timeout: 60
```

Comment:
OK - from tracing the usage of this timeout parameter in the salt code base, this is used for the timeout connecting to the salt master, and as a timeout when the salt master connects to a minion - this is not a job timeout.